### DISCUSSION TITLE
[IntelliSense] Add a feature to reload the IntelliSense without closing VSCode

### DISCUSSION THREAD

**Discussion by blopez2010** [tailwindlabs/tailwindcss#11219] [Ideas]
It is always beneficial not to close the entire VSCode instances if the TailwindCSS config file changes. Right now, If I add a new screen in the config list, I must close the VSCode instance to be able to hover the style and display the helper, which is so annoying!

Some extensions have an option item while doing `Ctrl + P` or `Cmd + P` to reload the extension on demand.

**Comment by rjfranc0** [comment]
Does someone actually working on it ?
